yjit/bindgen/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "yjit-bindgen"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Are we relaxing the "no crates from crates.io" thing? Or is it more that it's a pre-build step and we're checking in the results?